packages/cinterop/src/jvm/CMakeLists.txt

```
@@ -10,6 +10,9 @@ set(CMAKE_CXX_STANDARD 17)
 if (CMAKE_SYSTEM_NAME MATCHES "^Windows")
     MESSAGE("Building JNI for Windows")
     set(CMAKE_CXX_STANDARD_REQUIRED ON)
+    if (NOT WINDOWS_STORE)
+        set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
+    endif()
```

```suggestion
    set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
```
`WINDOWS_STORE` is enabled for UWP builds, which you don't do, so you don't need to worry about that condition - it will always be false in your builds.

---

Thanks 👍